abstract_specification.md

```diff
@@ -650,6 +656,12 @@ Examples, using a pseudo-syntax:
 
 ## Platform-Specific Configuration
 
+SOMA includes provisions for two separate ways to provide platform-specific configuration data to objects and operations.
+For configuration settings that affect a single operation, SOMA functions include a `platform_config` parameter, which can be used on a call-by-call basis to change implementation-specific behavior for that call.
+For configuration settings whose scope affects many operations, SOMA objects expose a `context` field that contains implementation-defined data that can be shared over an entire analysis session.
```

Comment:
We might want to also mention the parent/child inheritance of the context here, to introduce the idea before it is detailed below. Something like the below. I ended up editing both lines (platform_config and context), for consistency.

```suggestion
All SOMA operations include a `platform_config` parameter that allow a caller to provide implementation-specific configuration settings that affect the behavior of an operation on per-call basis.
All SOMA objects expose a `context` field that contain implementation-specific configuration settings that affect all operations called on a SOMA object and any of its children, and that are scoped to the life of these objects.
```

Reply:
This is a good idea. It should also become clearer when we specify the object lifecycle methods in a soon-to-come change.

Reply:
I also added some language here to note that context can be shared across more than just a single parent/child set.